### PR TITLE
Fixes #432 - Give common problem types as a choice for the user

### LINKED ISSUE #432: Greatest hits radio select
For bug reporting, populate a short list of most-submitted issues so it's easier to select what's going wrong.

As a mobile bug reporter
When I report a bug
Then I want it to be easier to select what's wrong
So I can report it faster.

### PR DESCRIPTION
**Pull request by miketaylr**
I deployed this to https://staging.webcompat.com so you can see it live and report test bugs, etc.

r? @magsout for CSS/HTML/JS bits.
r? @karlcow for Python bits.

### CODE REVIEW

**Comment by magsout** [comment]
:+1: 

lgtm..I just have to change some parts of the css, but one can merge before.

File: webcompat/form.py [hunk 1]
```diff
@@ -16,18 +16,22 @@
 from wtforms import TextAreaField
 from wtforms.validators import Length
 from wtforms.validators import Optional
-from wtforms.validators import Required
+from wtforms.validators import InputRequired
```
Comment (karlcow) [inline webcompat/form.py]: Alphabetical order. Nitpicking. so before "Length"

File: webcompat/form.py [hunk 2]
```diff
@@ -41,19 +45,16 @@
...
     username = StringField(u'Username',
                            [Length(max=0, message=username_message)])
-    description = TextAreaField(u'How can we replicate this?', [Optional()],
+    description = TextAreaField(u'Give more details', [Optional()],
                                 default=desc_default)
-    site_owner = RadioField(u'Is this your website?', [Optional()],
-                            choices=owner_choices)
-    problem_category = RadioField(u'What seems to be the trouble?',
-                                  [Optional()], choices=problem_choices)
+    problem_category = RadioField(u'What seems to be the trouble?*',
```
Comment (karlcow) [inline webcompat/form.py]: Typo? `?*`

**Comment by karlcow** [comment]
The python looks good. :+1: Just fix the small things and automatic r+ after this.

File: webcompat/form.py (hunk 1 above)
Comment (miketaylr) [inline webcompat/form.py]: Good catch.

File: webcompat/form.py (hunk 2 above)
Comment (miketaylr) [inline webcompat/form.py]: Nah, we have the `*` to indicate required (Currently on our Problem in 5 words* and Site URL* labels as wel. But now I notice we don't explain the * anywhere (maybe we don't need to?)

**Comment by miketaylr** [comment]
Okey doke, let's pull the trigger. :gun:

**Pull request merged**